Amalgam8 source code path

The instructions to build Amalgam8 components from source code says such code will be in $GOPATH/src/github.com/amalgam8/amalgam8 once you start up the vagrant vm. However, the source code is located in $GOPATH/src/github.com/amalgam8.
Hence, the building scripts fail with the following:

vagrant@vagrant-ubuntu-trusty-64:~/sandbox/src/github.com/amalgam8$ ./testing/build-scripts/build-controller.sh
+++ dirname ./testing/build-scripts/build-controller.sh
++ cd ./testing/build-scripts
++ pwd
- SCRIPTDIR=/home/vagrant/sandbox/src/github.com/amalgam8/testing/build-scripts
- MAKEDIR=/home/vagrant/sandbox/src/github.com/amalgam8/testing/build-scripts/../../
- make -C /home/vagrant/sandbox/src/github.com/amalgam8/testing/build-scripts/../../ build.controller GOOS=linux GOARCH=amd64
  make: Entering directory `/home/vagrant/sandbox/src/github.com/amalgam8'
  --> building controller
  cmd/controller/main.go:23:2: cannot find package "github.com/amalgam8/amalgam8/controller/api" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/api (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/api (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/api (from $GOPATH)
  cmd/controller/main.go:24:2: cannot find package "github.com/amalgam8/amalgam8/controller/config" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/config (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/config (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/config (from $GOPATH)
  cmd/controller/main.go:25:2: cannot find package "github.com/amalgam8/amalgam8/controller/metrics" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/metrics (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/metrics (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/metrics (from $GOPATH)
  cmd/controller/main.go:26:2: cannot find package "github.com/amalgam8/amalgam8/controller/middleware" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/middleware (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/middleware (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/middleware (from $GOPATH)
  cmd/controller/main.go:27:2: cannot find package "github.com/amalgam8/amalgam8/controller/rules" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/rules (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/rules (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/rules (from $GOPATH)
  cmd/controller/main.go:30:2: cannot find package "github.com/amalgam8/amalgam8/controller/util/i18n" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/controller/util/i18n (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/controller/util/i18n (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/controller/util/i18n (from $GOPATH)
  cmd/controller/main.go:28:2: cannot find package "github.com/amalgam8/amalgam8/pkg/auth" in any of:
  /home/vagrant/sandbox/src/github.com/amalgam8/vendor/github.com/amalgam8/amalgam8/pkg/auth (vendor tree)
  /usr/local/go/src/github.com/amalgam8/amalgam8/pkg/auth (from $GOROOT)
  /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8/pkg/auth (from $GOPATH)
  make: *** [build.controller] Error 1
  make: Leaving directory`/home/vagrant/sandbox/src/github.com/amalgam8'
- STATUS=2
- '[' 2 -ne 0 ']'
- echo -e '\n**********_\nFAILED: make failed for controller.\n**_********\n'

---

FAILED: make failed for controller.

---
- exit 2

--- a/devel/Vagrantfile
+++ b/devel/Vagrantfile
@@ -47,8 +47,8 @@ if ! grep -Fq "/usr/local/go/bin" /home/vagrant/.profile; then
 	echo 'export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin' >> /home/vagrant/.profile
 fi
 mkdir -p /home/vagrant/sandbox/src/github.com/amalgam8
-git clone https://github.com/amalgam8/amalgam8 /home/vagrant/sandbox/src/github.com/amalgam8/
-git clone https://github.com/amalgam8/a8ctl /home/vagrant/sandbox/src/github.com/amalgam8/
+git clone https://github.com/amalgam8/amalgam8 /home/vagrant/sandbox/src/github.com/amalgam8/amalgam8
+git clone https://github.com/amalgam8/a8ctl /home/vagrant/sandbox/src/github.com/amalgam8/a8ctl
 
 chown -R vagrant:vagrant /home/vagrant/sandbox
 SCRIPT

